scripts/git-all.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+PS3='git command to run for all templates: '
```

Comment:
I've been trying to run this locally and I can't figure out how to clone the submodules using this script. I've tried checkout, but it then prompts for a branch. I guess I use the same branch that I'm currently using?

![image](https://user-images.githubusercontent.com/1391619/79983326-9c819700-8475-11ea-9872-5819d8470d3a.png)


Also, I think you should abandon if they don't select an appropriate option since this is whats happening now.
![image](https://user-images.githubusercontent.com/1391619/79983269-84117c80-8475-11ea-8dfa-6483e755c178.png)

Reply:
> I've been trying to run this locally and I can't figure out how to clone the submodules using this script.

You don't need to use this script to clone the submodules, you can run `git submodule update --init --recursive` in the main repo. I think it should get all the submodules.

> Also, I think you should abandon if they don't select an appropriate option since this is whats happening now.

Ah yes I'll add this.

Reply:
Ok that makes sense. I thought originally this was tooling for getting setup locally.